bedrock/products/migrations/0005_monitorcalltoactionsnippet_monitorarticleindexpage_and_more.py

```diff
@@ -0,0 +1,130 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Generated by Django 4.2.20 on 2025-03-13 03:39
+
+import uuid
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+import wagtail.fields
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("cms", "0004_svgimage"),
+        ("wagtailcore", "0095_alter_page_title"),
```

Comment:
```suggestion
```
To get around this CI build error
> 7.107 django.db.migrations.exceptions.NodeNotFoundError: Migration cms.0004_svgimage dependencies reference nonexistent parent node ('wagtailcore', '0095_alter_page_title')

This looks like you may have run the migration against a newer version of Wagtail than in requierements, perhaps?

Reply:
I think I solved this but the tests are still failing if you could please look in the morning. 🥺